src/test/java/com/kodilla/ecommercee/domain/UserTestSuite.java

```diff
@@ -123,7 +123,7 @@ public void testBlockUser() {
     public void shouldCreateRelationWithOrder() {
         //Given
         User user = new User("user 1", "user@mail.pl", "PL");
-        Order order = new Order(Order.OrderStatus.UNPAID);
+        Order order = new Order(OrderStatus.UNPAID);
```

Comment:
ogólnie przyczepiam się tylko do używania get() z optionala, bo z tego co wiem nie chroni przed nullPointer... i lepiej go nie uzywać. Ale to tak tylko po to, żeby się do czegoś przyczepić :)

Reply:
.get() wyciąga obiekt z optionala kiedy jest potrzebny obiekt a nie optional

Reply:
a jeśli optional jest pusty, to rzuca NoSuchElementException, anie nullPointer... , pomyliłam się. W sumie chyba to będzie to samo co rzucane wyjątki przez nas